### PR TITLE
Chore/setup mocha

### PR DESCRIPTION
**Pull request by andela-kakpobome**
### What this PR does?
- upgrade the project routing system to fully leverage express
- make endpoints RESTful, start with creating a Fingerprint resource
- migrate current routes to Fingerprints resource
- correct error codes for invalid client requests from 500 to 400
- set up the Mocha testing framework
- include test spec

### CODE REVIEW

File: api/node-echoprint-server/specs/fingerprints.spec.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const should = require('should');
+const request = require('supertest');
+
+describe('Fingerprint Resource',function () {
+
+  it('should pass this test',function () {
+
+    //(5).should.be.exactly(5).and.be.a.Number();
```
Comment (makinwab) [inline api/node-echoprint-server/specs/fingerprints.spec.js]: Please remove commented out code.

Comment (andela-kakpobome) [reply api/node-echoprint-server/specs/fingerprints.spec.js]: yes ma'am!

**Comment by makinwab** [comment]
@andela-kakpobome the changes here seem to be in [PR 5](https://github.com/andela-kakpobome/hazaam/pull/5).

**Comment by andela-kakpobome** [comment]
@andela-bmakinwa remember I had to branch off the [PR5](https://github.com/andela-kakpobome/hazaam/pull/5) because it hadn't been merged yet.

**Comment by makinwab** [comment]
Oh okay.

We should have branched off the development branch actually. Or ensured that PR5 was merged.

We may need to close one of the PRs and resolve the conflicts that exist on this branch.

@andela-kakpobome

**Comment by makinwab** [comment]
There are still merge conflicts @andela-kakpobome.

**Pull request merged**